extensions/wikia/GlobalShortcuts/styles/GlobalShortcutsSearch.scss

```diff
@@ -0,0 +1,141 @@
+@import "skins/shared/color";
+
+#GlobalShortcutsSearch.global-shortcuts-search {
+	font-size: 14px;
+
+	border: 0;
+	height: 400px;
+	> header {
+		display: none;
+	}
+
+	> section {
+		height: 100%;
+		overflow: auto;
+		.col {
+			box-sizing: border-box;
+			float: left;
+			padding: 0 25px 0 0;
+			width: 50%;
+
+			&:nth-child(2n) {
+				padding: 0 0 0 25px;
+			}
+		}
+		.full-width {
+			box-sizing: border-box;
+		    position: relative;
+			clear: both;
+		}
+		#global_shortcuts_search_field {
+			background-image: url('/extensions/wikia/GlobalShortcuts/images/keyboard.svg')!important; /* inline */
+			background-position: 0px -9px;
+			background-repeat: no-repeat;
+			background-size: 33px;
+			border: 0;
+			box-sizing: border-box;
+			outline: 0;
+			padding: 8px 8px 8px 37px;
+			width: 100%;
+		}
+	}
+	
+	footer {
+		display: none;
+	}
+
+	h2 {
+		border-bottom: 1px solid #cacaca;
+		margin-bottom: 20px;
+	}
+	.key-list {
+		list-style: none;
+		overflow: hidden;
+	}
+  	.key-combination {
+	  	margin-bottom: 8px;
+		width: 50%;
+		line-height: 22px;
+	}
+	.key-combination, .key-combination-in-suggestions {
+	  	float: right;
+		text-align: left;
+	}
+	.key {
```

Comment:
I see that some styles are duplicated in these scss files. Maybe we can extract it and just import?